src/tests/Logout.test.js

```diff
@@ -0,0 +1,65 @@
+import '@testing-library/jest-dom/extend-expect';
+import { render } from "@testing-library/react";
+import { screen, waitFor } from '@testing-library/dom';
+import {BrowserRouter as Router} from 'react-router-dom';
+import user from '@testing-library/user-event'
+
+import Logout from "../components/Logout";
+
+describe('Logout tests', () => {
+    const logoutBtn = jest.fn();
+    const yesModalBtn = jest.fn();
+    const noModalBtn = jest.fn();
+    const mockedUsedNavigate = jest.fn();
+
+    jest.mock("react-router-dom", () => ({
+        ...jest.requireActual("react-router-dom"),
+        useNavigate: () => mockedUsedNavigate,
+    }));
+
+    it('Should logout with successfully and redirect to the login after', async () => {
```

Comment:
Should log out successfully after pressing the logout confirmation button*

Reply:
Done